### PR TITLE
Database connection added

### PR DESCRIPTION
**Pull request by MudassirQuraishi**

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const config = {
     production: {
         port: process.env.PROD_PORT || 8080,
         database: {
-            url: process.env.MONGODB_PROD_URI,
```
Comment (mudassirquraaishi) [inline config/config.js]: Change to PROD_URI

File: config/database.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            useNewUrlParser: true,
+            useUnifiedTopology: true,
+        });
+        const port = process.env.PORT || 8080;
+        const databaseStartTime = new Date().toLocaleString();
+        Logger.log("info", {
+            message: `MongoDB server successfully connected on ${databaseStartTime}`,
+        });
+    } catch (error) {
+        console.log(error);
```
Comment (mudassirquraaishi) [inline config/database.js]: Remove console.error()

File: util/logger.js [hunk 3]
```diff
@@ -109,6 +109,7 @@ process.on("unhandledRejection", (reason, promise) => {
 });
 
 process.on("uncaughtException", (error) => {
+    console.log(error);
```
Comment (mudassirquraaishi) [inline util/logger.js]: remove console.error()

**Review by mudassirquraaishi — CHANGES_REQUESTED** [review]
I've added comments please rectify those

File: config/database.js (hunk 2 above)
Comment (MudassirQuraishi) [reply config/database.js]: Resolved

File: util/logger.js (hunk 3 above)
Comment (MudassirQuraishi) [reply util/logger.js]: Resolved

**Review by mudassirquraaishi — APPROVED** [review]
Good to go

**Pull request merged**